adback-solution-to-adblock/includes/class-ad-back-activator.php

```diff
@@ -91,43 +115,30 @@ public static function activate()
             }
 
             $wpdb->insert(
-                $table_name,
+                $table_name_token,
                 [
-                    "id"            => "1",
+                    "id"            => $blogId,
                     "access_token"  => $accessToken,
                     "refresh_token" => $refreshToken
                 ]
             );
 
-            $savedToken = $wpdb->get_row("SELECT * FROM " . $table_name . " WHERE id = 1");
+            $savedToken = $wpdb->get_row("SELECT * FROM " . $table_name_token . " WHERE id = ".$blogId);
         }
 
-        //create myinfo table
-        $table_name = $wpdb->prefix . 'adback_myinfo';
-
-        $sql = "CREATE TABLE ".$table_name." (
-            `id` mediumint(9) NOT NULL,
-            `myinfo` text DEFAULT '' NOT NULL,
-            `domain` text DEFAULT '' NOT NULL,
-            `update_time` DATETIME NULL,
-            UNIQUE KEY id (id)
-        ) ".$charset_collate.";";
-
-        dbDelta( $sql );
-
         $wpdb->insert(
-            $table_name,
+            $table_name_info,
             array(
-                "id" => "1",
+                "id" => $blogId,
```

Comment:
as you are creating one table per blog, I am not sure that you still needs to set the id to the `$blogId`